src/app/routes/_meditation.router.js

```diff
@@ -109,19 +109,59 @@ export default (app, router, io) => {
         await meditation.remove();
       }
 
+      let medStart = new Date(new Date().getTime());
+      let medEnd = new Date(new Date().getTime() + total * 60000);
+
+      // check if custom session start date/time was requested
+      if (req.body.start) {
+        let newStart = moment.utc(req.body.start).toDate();
+        let newEnd = moment.utc(req.body.start).add(total, 'minutes').toDate();
+
+        // check if date is valid
+        if (isNaN(newEnd.getTime()) || newEnd >= moment.utc().toDate()) {
```

Comment:
We should only allow to add times for the last 30 days or so. Otherwise people may fake old sessions to have better stats.